CGPs/cgp-0039/cgp_0036.json

```diff
@@ -0,0 +1,8 @@
+[
```

Comment:
the file is called cgp_0036.json but it's targeting cgp-0039. Is that the right number?

Reply:
typo, thanks